cli/up.go

```diff
@@ -0,0 +1,26 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	upCmd = &cobra.Command{
+		Use:   "up <group> [scale]",
+		Short: "Bring a group up",
+		Long:  "Submit a group, with an optional scale, and start it",
+		Run:   upRun,
+	}
+)
+
+func upRun(cmd *cobra.Command, args []string) {
+	submitRun(cmd, args)
```

Comment:
If there are already 2 slices submitted of 'mygroup', and you up 2 slices of 'mygroup', then you will end up with 4 running slices, as up == submit then start, meaning that the 2 previously submitted slices will also get started.

I think having starting all four slices is logically consistent. If we see that people find this confusing, we could iterate on it - for example, up could grumble if there is an existing group.